gen/templates/component/component-name.adb

```diff
@@ -943,12 +943,32 @@ package body Component.{{ name }} is
 
    not overriding procedure Process_Parameter_Update (Self : in out Base_Instance; Par_Update : in out Parameter_Update.T) is
       use Parameter_Operation_Type;
+      use Parameter_Validation_Status;
+      use Parameter_Update_Status;
       Status : Parameter_Update_Status.E := Parameter_Update_Status.Success;
    begin
       case Par_Update.Operation is
          when Stage =>
             -- Stage this parameter.
             Status := Self.Stage_Parameter (Par_Update.Param);
+         when Validate =>
+            -- Pass the staged parameters to the user defined validation function.
+            -- Note that parameter ranges have already been validated as part of staging
+            -- each parameter individually, this function is used for extended,
+            -- user-implemented validation.
+            if Base_Instance'Class (Self).Validate_Parameters_Action (
+{% for par in parameters %}
+               {{ par.name }} => Self.Staged_Parameters.Get_{{ par.name }}{{ "," if not loop.last }}
+{% endfor %}
+            ) /= Valid
+            then
+               -- The default return for this function always returns "Valid", so this
+               -- should be inaccessible.
```

Comment:
Remove this comment

Reply:
Addressed in [c61b1f5](https://github.com/lasp/adamant/pull/64/commits/c61b1f53a99257b70b15bbd0a22495ca6cccd935).